deploy/olm-catalog/nexus-operator/0.3.0/nexus-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -75,6 +75,13 @@ spec:
         path: image
         x-descriptors:
         - urn:alm:descriptor:io.kubernetes:image
+      - description: 'The image pull policy for the Nexus image. If left blank behavior
+          will be determined by the image tag (`Always` if "latest" and `IfNotPresent`
+          otherwise). Possible values: `Always`, `IfNotPresent` or `Never`.'
+        displayName: Image Pull Policy
+        path: imagePullPolicy
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:imagePullPolicy
```

Comment:
Oh, I didn't know we have this `x-descriptor` type. 
EDIT: it seems the operator-sdk is now smarter than me =D